feat: Document Custom GCP Billing Account feature (#1013)
meshcloud/meshcloud-docs: docs/meshstack.how-to.integrate-meshplatform-gcp-manually.md

Following the principle of least privilege, operators should remove the `billing.resourceAssociations.create` permisson from the custom role `meshfed-service` created in [meshfed-service IAM Role](#set-up-the-service-account-for-replication).

#### Optional: Using Different Billing Accounts for Different Google Cloud Projects

By default, meshStack will associate all created Google Cloud projects with the same billing account. If you want to use different
billing accounts for different projects, you can do so by doing the following:

1. Create a dedicated tag in meshStack to track custom billing account IDs. This can be a tag on the workspace, project or payment method.

> 2. Set the name of this tag in the Dhall configuration for `web.tags.tagNameUsedForGcpBillingAccount`. Ask your Customer Success representative for help here.

Author: Yeah you're right. I changed the docus

Following the principle of least privilege, operators should remove the `billing.resourceAssociations.create` permisson from the custom role `meshfed-service` created in [meshfed-service IAM Role](#set-up-the-service-account-for-replication).

#### Optional: Using Different Billing Accounts for Different Google Cloud Projects

By default, meshStack will associate all created Google Cloud projects with the same billing account. If you want to use different

> billing accounts for different projects, you can do so by doing the following:

Reviewer: ```suggestion
billing accounts for different projects, you can do so via the following steps:
```

d: Think that sounds a bit nicer
